to-do-list/src/components/TaskBox.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+
+interface Props {
+    data: Array<{ id: number, desc: string, done: boolean, save: boolean }>
+}
+
+interface SectionState {
+
+}
+
+class TaskBox extends Component<Props> {
+
+    render(): JSX.Element {
+
+        let taskElementlist
```

Comment:
```suggestion
        const taskElementlist = this.props.data.map~~
```

이 변수를 선언하는 16번 라인과 이 변수에 값을 할당하는 19번 라인, 그리고 사용하는 38번 라인 중에서 이 변수가 재할당 되는 경우는 존재하지 않습니다. 이런 경우에는 변수를 재할당하지 않도록 const로 선언하는 것이 좋습니다. 아무리 짧은 코드라도, 개발자가 실수할 여지를 주면 언젠가는 예상하지 않은 버그를 발생시킬 수 있습니다.